src/main/src/test/java/org/geoserver/catalog/impl/LocalWorkspaceLayersTest.java

```diff
@@ -59,6 +61,16 @@ public void testGroupLayerInWorkspace() {
         assertNotNull(catalog.getLayerGroupByName("localGroup"));
         assertEquals(1, catalog.getLayerGroups().size());
         assertEquals("localGroup", catalog.getLayerGroupByName("localGroup").prefixedName());
+        
+        GeoServer gs = getGeoServer();
+        SettingsInfo settings = gs.getFactory().createSettings();
+        settings.setLocalWorkspaceIncludesPrefix(true);
```

Comment:
Yes, there is a CheckBox in the workspace web-admin panel. Other similar wrap methods in the LocalWorkspaceCatalog class already use this flag. Without this fix you can get a GetCapabilites XML document with layers without prefix even with flag activated.